Firebase/Core/FIRVersion.m

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+
+#ifndef Firebase_VERSION
+#error "Firebase_VERSION is not defined: add -DFirebase_VERSION=... to the build invocation"
```

Comment:
@paulb777 Will this cause trouble for future build systems that don't use CocoaPods?

Reply:
@ryanwilson This should be fine. We already require any build system to add -D versions for multiple Firebase components.  This ifndef is a great way to catch it at build time.

Separately, we do often use #ifdef COCOAPODS in a binary way that will have to be reviewed with a third build system.

Reply:
Thanks!